src/main/java/thePackmaster/SpireAnniversary5Mod.java

```diff
@@ -506,8 +518,58 @@ public void receiveOnBattleStart(AbstractRoom room) {
         CLAW_SHARP_TRACKER = 0;
         combatExhausts = 0;
     }
-    
-	@Override
+
+    @Override
+    public void receiveOnPlayerTurnStart() {
```

Comment:
Please put this logic into another class and call it from here.
So for example.
`public void receiveOnPlayerTurnStart() {
Leprachaun.receieveOnPlayerTurnStart();
}`
The main mod file is getting a little crowded